### PR TITLE
Better code indentation and modules renaming 

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

File: src/ol/layer/Tile.js [hunk 1]
```diff
@@ -53,9 +53,7 @@ inherits(TileLayer, Layer);
  * @api
  */
 TileLayer.prototype.getPreload = function() {
-  return (
-  /** @type {number} */ this.get(_ol_layer_TileProperty_.PRELOAD)
-  );
+  return (/** @type {number} */ this.get(TileProperty.PRELOAD));
```
Comment (tschaub) [inline src/ol/layer/Tile.js]: These parens might have gotten messed up by the initial code transform.  Shouldn't this be like:
```js
return /** @type {number} */ (this.get(TileProperty.PRELOAD));
```

(I'll be happy when we get rid of these typecasts altogether.)

File: src/ol/layer/Tile.js [hunk 2]
```diff
@@ -86,9 +84,7 @@ TileLayer.prototype.setPreload = function(preload) {
  * @api
  */
 TileLayer.prototype.getUseInterimTilesOnError = function() {
-  return (
-  /** @type {boolean} */ this.get(_ol_layer_TileProperty_.USE_INTERIM_TILES_ON_ERROR)
-  );
+  return (/** @type {boolean} */ this.get(TileProperty.USE_INTERIM_TILES_ON_ERROR));
```
Comment (tschaub) [inline src/ol/layer/Tile.js]: Same here regarding the grouping.

File: src/ol/layer/VectorTile.js [hunk 3]
```diff
@@ -77,9 +75,7 @@ VectorTileLayer.prototype.getPreload = function() {
  * @api
  */
 VectorTileLayer.prototype.getUseInterimTilesOnError = function() {
-  return (
-  /** @type {boolean} */ this.get(_ol_layer_TileProperty_.USE_INTERIM_TILES_ON_ERROR)
-  );
+  return (/** @type {boolean} */ this.get(TileProperty.USE_INTERIM_TILES_ON_ERROR));
```
Comment (tschaub) [inline src/ol/layer/VectorTile.js]: Grouping.

File: src/ol/layer/Tile.js (hunk 1 above)
Comment (fredj) [reply src/ol/layer/Tile.js]: in a8f4348

**Pull request merged**